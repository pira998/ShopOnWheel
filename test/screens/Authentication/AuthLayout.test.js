import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../../../screens/Authentication/AuthLayout'

jest.mock('react-native-keyboard-aware-scroll-view', () => {
    const KeyboardAwareScrollView = ({ children }) => children;
    return { KeyboardAwareScrollView };
});
test('renders correctly', () => {
  const tree = renderer.create(<Layout />).toJSON();
  expect(tree).toMatchSnapshot();
});