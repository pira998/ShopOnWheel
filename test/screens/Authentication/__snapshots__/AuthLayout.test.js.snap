// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingVertical": 24,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <Image
      resizeMode="contain"
      source={1}
      style={
        Object {
          "height": 100,
          "width": 200,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "marginTop": -14,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "Poppins-Bold",
          "fontSize": 22,
          "lineHeight": 30,
          "textAlign": "center",
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#525C67",
          "fontFamily": "Poppins-Regular",
          "fontSize": 16,
          "lineHeight": 22,
          "marginTop": 8,
          "textAlign": "center",
        }
      }
    />
  </View>
</View>
`;
